tsl/src/remote/copy_fetcher.c

```diff
@@ -104,12 +107,11 @@ copy_fetcher_send_fetch_request(DataFetcher *df)
 		{
 			ereport(ERROR,
 					(errcode(ERRCODE_CONNECTION_FAILURE),
-					 errmsg("could not set single-row mode on connection to \"%s\"",
+					 errmsg("could not set COPY mode on connection to \"%s\"",
```

Comment:
The previous text was correct, it fails to set single-row mode here. COPY mode is set by sending the COPY query.